kubernetes/gitlab-auth.sh

```diff
@@ -131,7 +131,8 @@ fi
 echo "GitLab Token: $gitlabToken"
 
 k8sApi="${K8S_API:-https://127.0.0.1:443}"
-k8sNamespaces="${K8S_NAMESPACES:-default}"
+k8sNamespaces=($(echo ${K8S_NAMESPACES:-default} | tr "," " "))
+k8sContexts=($(echo ${K8S_CONTEXTS:-default} | tr "," " "))
 
```

Comment:
@targs08 we should add validation here that number of items in `k8sNamespaces` equals to number of items in `k8sContexts`. Otherwise UB is possible.